run_integration.sh

```diff
@@ -7,7 +7,7 @@ pushd $rootdir
 
 # Reset test harness
 rm -rf test-harness
-git clone --single-branch --branch develop https://github.com/algorand/algorand-sdk-testing.git test-harness
+git clone --single-branch --branch evan/rekey_tester https://github.com/algorand/algorand-sdk-testing.git test-harness
```

Comment:
Need to merge in `evan/rekey_tester` and change these back to develop

Reply:
Changed back to `develop`. No need to merge `evan/rekey_tester`, I think, it was a temp branch for modified testing